src/tests/integration/test_mysql.py

```diff
@@ -0,0 +1,90 @@
+import pandas as pd
+import pytest
+from sqlalchemy import create_engine
+
+# TODO: move postgreSQLConfig to individual file, shared with ci-integration-db
+databaseConfig = {
```

Comment:
let's simplify the tests by parametrizing over fixtures: https://github.com/ploomber/jupysql/issues/92#issuecomment-1416209770

Reply:
Refract in [Add: Generic dynamic update](https://github.com/ploomber/jupysql/pull/111/commits/3e609766850e2849238a8622648c096a20e1c127)

I abstract the `test_mysql.py` and `test_postgresql.py` and consolidate in one file [test_generic_db_opeations.py](https://github.com/ploomber/jupysql/pull/111/commits/3e609766850e2849238a8622648c096a20e1c127#diff-3c90104262a212ec645651cab58e4dc3a92b8f283795671f2ace3d2cd8b13555)

In the `test_generic_db_opeations.py`, It will get the dynamic fixture `ip_with_dynamic_db ` based on different database & run the test case.

I think we can use this as a basic framework to test the same behavior among different databases, for the feature only provided in individual databases, we can create another `test_[database_name].py` to cover it, although I haven't thought about what would be that now